server/chat-plugins/scavengers.ts

```diff
@@ -747,14 +891,14 @@ class ScavengerHunt extends Rooms.RoomGame {
 
 	onUpdateConnection() {}
 
-	onChatMessage(msg) {
-		let msgId = toID(msg);
+	onChatMessage(msg: string) {
+		let msgId = toID(msg) as string;
 
 		const commandMatch = msg.match(SCAVENGE_REGEX);
 		if (commandMatch) msgId = msgId.slice(toID(commandMatch[0]).length);
```

Comment:
If you're not going to use the return value of `match`/`exec`, you should be using `test` for performance:

```suggestion
		if (ACCIDENTAL_LEAKS.test(msg)) {
			msgId = msgId.slice(toID(commandMatch[0]).length);
		}
```